interview-questions.md

```diff
@@ -6,62 +6,50 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-
 1. What is `this` in JavaScript?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer: an object
 
+Researched answer: an object carrying out a specific piece of code
 
 2. What is React? Why would you use it?
 
-  Your answer:
-
-  Researched answer:
-
+Your answer: A built in library where users can create applications
 
+Researched answer: An open-source library within JavaScript that allows you to build apps
```

Comment:
Could you go into more detail on why you would want to use React to build an app?  What are the benefits of using React?